### PR TITLE
Version bump for RHMAP-14611

### PR DESCRIPTION
**Pull request by thailekha**
Related of:

JIRA: https://issues.jboss.org/browse/RHMAP-14611
JIRA: https://issues.jboss.org/browse/RHMAP-13787
JIRA: https://issues.jboss.org/browse/FH-3422
JIRA: https://issues.jboss.org/browse/FH-2771

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "template-config",
-  "version": "4.12.6",
+  "version": "4.13.6",
```
Comment (pb82) [inline package.json]: shouldn't it be 4.13.0?

File: sonar-project.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
 sonar.projectKey=fh-template-apps
 sonar.projectName=fh-template-apps-nightly-master
-sonar.projectVersion=4.12.6
+sonar.projectVersion=4.13.6
```
Comment (pb82) [inline sonar-project.properties]: same as above

File: package.json (hunk 1 above)
Comment (thailekha) [inline package.json]: It had been 4.13.0, then I made some changes and bumped the patch a few times. I used `commit --amend` so the bumps are not shown here. I'll change it back to 4.13.0 if you'd like

**Comment by thailekha** [comment]
@pb82 does it look better now?

**Pull request merged**